env.list.example

```diff
@@ -6,7 +6,7 @@ INITSYSTEM=off
 password=
 proxy_ssh_port=22
 requests_verify=False
-RESINRC_PROXY_URL=balena-devices.com
-RESINRC_RESIN_URL=balena-cloud.com
+BALENARC_PROXY_URL=balena-devices.com
+BALENARC_RESIN_URL=balena-cloud.com
```

Comment:
Should that be `BALENARC_BALENA_URL` ?  :shrug:

Reply:
Thank you.